src/ResourceManager/RecoveryServices.SiteRecovery/Commands.RecoveryServices.SiteRecovery/RecoveryPlan/UpdateAzureRmRecoveryServicesAsrRecoveryPlan.cs

```diff
@@ -148,6 +148,20 @@ public override void ExecuteSiteRecoveryCmdlet()
                                             VmId = ((HyperVReplicaBlueReplicationDetails)item
                                                 .Properties.ProviderSpecificDetails).VmId;
                                         }
+                                        else if (item.Properties.ProviderSpecificDetails
+                                                .GetType() ==
```

Comment:
@viverm nit: for the lines where `.GetType()` is by itself, can you tab it to be consistent with the other instances of it above

It looks like this is also the case in `NewAzureRmRecoveryServicesAsrRecoveryPlan.cs`

Reply:
ok